CompilerSource/compiler/compile.cpp

```diff
@@ -150,10 +150,22 @@ inline void write_exe_info(const std::string codegen_directory, const EnigmaStru
 
 #include "System/builtins.h"
 
-dllexport int compileEGMf(EnigmaStruct *es, const char* exe_filename, int mode) {
+static inline int compileEGMf_Raw(EnigmaStruct *es, const char* exe_filename, int mode) {
```

Comment:
Ok, I'll remove it.

Reply:
I took care of it in 30c48d305a44626495438967e57de7aef8ae4275. This is actually better because now it means that if this pr were merged, it would not break anybody on the existing portable ZIP setup if they tried to pull. It means this pr does not add any new dependencies for them unless they try to manually build the protos.